python/GafferSceneUI/_SceneViewInspector.py

```diff
@@ -178,11 +181,11 @@ def __init__( self, sceneView ) :
 	def setVisible( self, visible ) :
 
 		# The view.inspector.visible plug is the authority on whether the
-		# inspector should be visible. As we're managed by the layout,
-		# setVisible will be called at various times during the lifecycle of
-		# the parent view. We don't want to be appearing if the user has turned
-		# us off.
-
+		# inspector should be visible. setVisible may be called by our parent
+		# layout as its visibility changes. We need to ensure we don't appear
+		# the user has disabled us. Considering the plug state here allows
```

Comment:
the user -> if the user

Reply:
I don't know what is wrong with my brain, sorry, fixed [here](https://github.com/GafferHQ/gaffer/pull/3832/commits/540625f6b4452ad4f3dec615e43566cadf0b2bf8).